### PR TITLE
fix website address

### PR DESCRIPTION
**Pull request by returnofthelambda**
Website address for The AVR-Rust Book in README.md is still set to https://book.avr-rust.com which no longer exists. Site is now hosted at https://book.avr-rust.org

When searching for AVR and Rust, this was the first page in my results. Submitting this pull request to help others find the correct resource.

### CODE REVIEW

**Comment by stappersg** [comment]
On Tue, Apr 23, 2024 at 05:41:02AM -0700, Bill Lillyman wrote:
> Website address for The AVR-Rust Book in README.md is still set to
> https://book.avr-rust.com which no longer exists. Site is now hosted
> at https://book.avr-rust.org

OK

 
> When searching for AVR and Rust, this was the first page in my
> results. Submitting this pull request to help others find the correct
> resource.

True.

However the commit has only a commit summary.

Give it a complete commit message.


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by returnofthelambda** [comment]
Added more information to the commit. Please advise if there is anything else to add or change. Thanks.
